src/crepe/system/InputSystem.h

```diff
@@ -37,8 +38,8 @@ class InputSystem : public System {
 	//! Stores the last mouse button pressed.
 	MouseButton last_mouse_button = MouseButton::NONE;
 
-	//! The maximum allowable distance between mouse down and mouse up to register as a click.
-	const int click_tolerance = 5;
+	//! The maximum allowable distance between mouse down and mouse up to register as a click. This can be changed using the Config.
+	int click_tolerance = Config::get_instance().click_tolerance.tolerance;
```

Comment:
Door deze property zo op te slaan inplaats van zo laat mogelijk opvragen waar je hem pas nodig hebt maak je het effectief een default waarde, omdat aanpassingen na de InputSystem initialisatie niet meer gelezen zullen worden. Wil je dit aanpassen of duidelijker documenteren in `src/crepe/api/Config.h`?